### ISSUE TITLE
[🐛 Bug]: Selenium manager not downloading latest stable CfT version

### ISSUE THREAD

**Issue by jvanderwoude80** [SeleniumHQ/selenium#12490]
### What happened?

Tried using selenium manager in .NET to always download the latest stable CfT version by providing `options.BrowserVersion = "stable"`. But then the already installed version 114 is used instead of latest stable 115. Tried the same with the selenium manager binary for troubleshooting and this showed up in the debug log. 

Seems to me this line indicates a bug: `Online stable chrome version (115) is the same as the one installed in the system`, it states 115 equals 114 and then goes on using 114??

FYI: other channels as 'dev' or 'beta' are working as expected, 116 and 117 of CfT are downloaded. 

```
C:\Temp>selenium-manager.exe --debug --browser chrome --browser-version stable
DEBUG   Checking chromedriver in PATH
DEBUG   Running command: chromedriver --version
DEBUG   Output: ""
DEBUG   chromedriver not found in PATH
DEBUG   chrome detected at C:\Program Files\Google\Chrome\Application\chrome.exe
DEBUG   Using shell command to find out chrome version
DEBUG   Running command: wmic datafile where name='C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe' get Version /value
DEBUG   Output: "\r\r\n\r\r\nVersion=114.0.5735.248\r\r\n\r\r\n\r\r\n\r"
DEBUG   Detected browser: chrome 114.0.5735.248
DEBUG   Reading metadata from https://googlechromelabs.github.io/chrome-for-testing/last-known-good-versions-with-downloads.json
DEBUG   Online stable chrome version (115) is the same as the one installed in the system
DEBUG   Required driver: chromedriver 114.0.5735.90
DEBUG   chromedriver 114.0.5735.90 already in the cache
INFO    Driver path: C:\Users\<myusername>\.cache\selenium\chromedriver\win64\114.0.5735.90\chromedriver.exe
INFO    Browser path: C:\Program Files\Google\Chrome\Application\chrome.exe
```

### How can we reproduce the issue?

```shell
selenium-manager.exe --debug --browser chrome --browser-version stable
```

### Relevant log output

```shell
See issue
```

### Operating System

Windows 10

### Selenium version

.NET 4.11

### What are the browser(s) and version(s) where you see this issue?

Chrome 114 / 115

### What are the browser driver(s) and version(s) where you see this issue?

N/A

**Comment by diemol** [comment]
This is a duplicate of #12468. It is fixed in `trunk` and will be available in the next release (latest in ~2.5 weeks).

**Comment by jvanderwoude80** [comment]
Thanks, I overlooked the existing bug. Awaiting the fix.
